commands/drive/teleop/swerve/XboxSwerveDrive.java

```diff
@@ -58,8 +61,8 @@ public void execute() {
             thetaVel = rotPIDController.calculateRotationSpeed(driveBase.getGyroscopeRotation(),
                     desiredHeading);
         }
-
-        driveBase.drive(ChassisSpeeds.fromFieldRelativeSpeeds(xVel, yVel, thetaVel,
+        double speedMult= SmartDashboard.getNumber("Swerve Speed", 0.66);
```

Comment:
what is this?

Reply:
bump

Reply:
Temporary speed control code. Removed. Look above for more details